### PR TITLE
feat: able to list rayclusters configured in arcadia config

### LINKED ISSUE #510: able to use ray in worker
Based on this request https://github.com/kubeagi/ops-console/issues/136 
- [x] #511
- [ ] #524

### LINKED ISSUE #511: able to list ray clusters

### PR DESCRIPTION
**Pull request by bjwswang**
#### What type of PR is this?
/kind feat

#### Which issue(s) this PR fixes

Fixes #511 

#### Special notes for your reviewer

We only need to list ray clusters that have been confiugred in our `arcadia config` at  https://github.com/kubeagi/arcadia/blob/main/pkg/config/config_type.go#L38

### CODE REVIEW

File: apiserver/pkg/ray/raycluster.go [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+		results = append(results, &generated.RayCluster{
+			Index:         index,
+			Name:          cluster.Name,
+			HeadAddress:   &cluster.HeadAddress,
+			DashboardHost: &cluster.DashboardHost,
+			PythonVersion: &cluster.PythonVersion,
+		})
+	}
+
+	return &generated.PaginatedResult{
```
Comment (0xff-dev) [inline apiserver/pkg/ray/raycluster.go:53]: Does `RayCluster` need paging?

File: apiserver/graph/schema/worker.graphqls [hunk 2]
```diff
@@ -197,7 +197,7 @@ input ListWorkerInput {
     labelSelector: String
     """字段选择器"""
     fieldSelector: String
-    """
+    """
```
Comment (Abirdcfly) [inline apiserver/graph/schema/worker.graphqls:200]: nit: revert this line

**Review by Abirdcfly — APPROVED** [review]

File: apiserver/pkg/ray/raycluster.go (hunk 1 above)
Comment (bjwswang) [reply apiserver/pkg/ray/raycluster.go:53]: I don't think so. They only need the whole list of the ray clusters .

File: apiserver/pkg/ray/raycluster.go [hunk 3]
```diff
@@ -0,0 +1,57 @@
+/*
+Copyright 2023 KubeAGI.
```
Comment (Abirdcfly) [inline apiserver/pkg/ray/raycluster.go:2]: Nit: 2024 😂

**Pull request merged**